Objects/longobject.c

```diff
@@ -4666,6 +4716,22 @@ long_lshift1(PyLongObject *a, Py_ssize_t wordshift, digit remshift)
         stwodigits x = m < 0 ? -(-m << remshift) : m << remshift;
         return _PyLong_FromSTwoDigits(x);
     }
+    if (IS_MEDIUM_VALUE(a) && a->ob_digit[0] == (digit)1) {
+        /* shifting is guaranteed positive, so there is
+           no need for filtering less than 0 shifting unlike
+           long_pow */
+        /* if statement for checking greater than 0 shifts */
+        if (wordshift || remshift) {
+            z = _PyLong_NewFillZero(wordshift + 1);
+            if (z == NULL)
+                return NULL;
+            z->ob_digit[wordshift] = (digit)(Py_SIZE(a) << remshift);
```

Comment:
I think this is incorrect.

Before this change:
```
>>> (-1) << 100
-1267650600228229401496703205376
```

After:
```
>>> (-1) << 100
5316910715489063263385826744418172928
```

Reply:
I'd suggest making sure the tests (`./python -m test test_long`) pass locally before committing to your remote repo.

Reply:
I make the edits on the GitHub website itself.

Reply:
Fixed.

Reply:
> I make the edits on the GitHub website itself.

If you have the opportunity and an OS that can install software other than a browser, getting a local development environment set up might make things easier. There are helpful directions here https://devguide.python.org/setup/